src/NServiceBus.AwsLambda.SQS/TransportMessage.cs

```diff
@@ -14,7 +11,7 @@ public TransportMessage()
         {
         }
 
-        public TransportMessage(OutgoingMessage outgoingMessage, List<DeliveryConstraint> deliveryConstraints)
+        public TransportMessage(OutgoingMessage outgoingMessage, DispatchProperties deliveryConstraints)
```

Comment:
can we rename the argument name to `properties` so that it is aligned with the transport's type?